package.json

```diff
@@ -36,7 +36,7 @@
     "set.js": "^0.3.2",
     "styled-components": "^5.3.3",
     "use-wallet": "^0.11.0",
-    "web3": "^1.5.2",
+    "web3": "^1.6.0",
```

Comment:
LGTM! 

[Off topic] Is there any interest in switching from web3 to ethers which has much stronger TypeScript support?

Reply:
For me personally? Yes, I'd like to on the rewrite

Reply:
Same, I'm a big fan of others